lib/kwtool_web/templates/auth/form.html.eex

```diff
@@ -0,0 +1,35 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+  <div class="form-group">
+    <%= label f, :email %>
+    <%= email_input f, :email, required: true, class: "form-control", placeholder: "Email" %>
+    <%= error_tag f, :email %>
+  </div>
+  <div class="form-group">
+    <%= label f, :full_name %>
+    <%= text_input f, :full_name, required: true, class: "form-control", placeholder: "Full name" %>
+    <%= error_tag f, :full_name %>
+  </div>
+  <div class="form-group">
+    <%= label f, :company %>
+    <%= text_input f, :company, class: "form-control", placeholder: "Company" %>
+    <%= error_tag f, :company %>
+  </div>
+  <div class="form-group">
+    <%= label f, :password %>
+    <%= password_input f, :password, minlength: 5, required: true, class: "form-control", placeholder: "Password" %>
```

Comment:
The password minimum here is not match validation in the schema 🙌

Reply:
updated.